src/os/bluestore/bluestore_tool.cc

```diff
@@ -346,7 +346,7 @@ int main(int argc, char **argv)
         "reshard, "
         "show-sharding, "
        	"trim, "
-        "downgrade-wal-v1")
+        "migrate-wal-to-v1")
```

Comment:
We've  used 'migrate' keyword for data migration already. IMO better not to use it for a completely different purpose. What's about 'switch-wal-to-v1' or 'set-v1-wal'?

Reply:
"migrate" denotes some process, "switch"/"set" suggest some atomic action.
Isn't a process that requires reading data from one place and saving it on new place, changing format in the way a perfect fit for "migrate"?

Reply:
My primary point is that "migrate" keyword is already in use for BlyeFS data migation between volumes. So IMO having "migrate-wal..." looks a bit confusing. Other than that it's fine..

Reply:
What's about "downgrade-wal-to-v1"?

Reply:
or "revert-wal-to-v1"?